src/ngraph/frontend/onnx_import/utils/reshape.hpp

```diff
@@ -22,13 +22,31 @@ namespace ngraph
 {
     namespace onnx_import
     {
+        namespace utils
+        {
+            /**
+             * @brief Flatten the input tensor into a 2D matrix.
+             *
+             * @details Flattening happens at axis specified by 'axis' attribute.
+             *  First dimension of output tensor is the product of [d_0, ... d_{axis-1}] dimensions of input tensor.
+             *  The last dimension is the product of the rest of input tensor dimensions: [d_{axis}, ..., d_n]
+             *
+             * @param node The tensor to be flattened.
+             * @param axis The axis dividing shape.
+             *
+             * @return The new node being a 2D matrix representing flattened input node.
+             */
+            std::shared_ptr<ngraph::Node> flatten(const std::shared_ptr<ngraph::Node>& node,
+                                                  int axis);
+        } // namespace utils
+
         /**
          * @brief Permute axes according to specified axes_order parameter.
          *
          * @param node The node which axes we want to permute.
          * @param axes_order The permutation of node tensor axes.
          *
-         * @return: New node with permuted axes.
+         * @return New node with permuted axes.
```

Comment:
Doc style.

Reply:
Only in this place return had a colon before description, so I removed it.

Reply:
Adam is refering to use `\` instead of `@`